cscs-checks/mch/collectives_halo.py

```diff
@@ -84,6 +84,7 @@ def __init__(self, bench_reference):
         }
 
         self.maintainers = ['AJ', 'MKr']
+        self.strict_check = False
```

Comment:
I think you've added the `strict_check` into the wrong test. This should go into the `halo_cell_exchange.py` instead.